lib/ht/search_client/test.rb

```diff
@@ -38,11 +38,11 @@ def stay_class?
       self.class.to_s =~ /Stay/
     end
 
-    def mocked_results(properties)
+    def mocked_results(properties, total)
       {
         page:     params.fetch(:page),
         per_page: params.fetch(:per_page),
-        total:    properties.length,
+        total:    total,
```

Comment:
Then I would say that I simply default to `properties.size` like it was before.
Everything else is left to the caller.
I pushed the change